src/install/java/frameworkInstallation/springboot.mdx

```diff
@@ -10,5 +10,5 @@ descriptionText: To complete your installation, follow these framework-specific
 To pass the `-javaagent` argument on Spring Boot, add it to the command line in which you start your app. Make sure to add it before the `-jar` argument:
 
 ```bash
-java -javaagent:/full/path/to/newrelic.jar -jar app.jar
+java -javaagent:/full/path/to/newrelic.jar -jar /full/path/to/{YourApplicationName}.jar
```

Comment:
Try instead:

```
java -javaagent:/path/to/newrelic.jar -jar /path/to/{YourApplicationName}.jar
```

Customers may not necessarily need the full path but as long as the path reaches your respective jars, it should work.